<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PingCheckerStats</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Otwórz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateExcelFileButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generuj arkusz</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="minEventsSpinBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxEventDifferenceSecSpinBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="minEventsLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimana liczba zdarzeń</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxEventDiferenceSecLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maksymalna różnica między zdarzeniami [sec]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
